students/M_Sunday/lesson03/list_lab.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/python
+
+# lesson 03 - list lab exercise
+
+
+def list_contents(list_, id_=1):
+    print("\nSeries List Includes the Following Fruits:")
+    for j, item_ in enumerate(list_):
+        if id_ == 1:
+            print("  - {}".format(item_))
+
```

Comment:
There's no need to call `enumerate()` in this loop since you aren't referencing the index of the list in the loop, so I believe this function can be rewritten as:
```
def list_contents(list_):  
    print("\nSeries List Includes the Following Fruits:")
    for item_ in list_:
        print("  - {}".format(item_))
```